### DISCUSSION TITLE
Deployed Elastalert over kubernetes manually

### DISCUSSION THREAD

**Discussion by Raks2110** [jertel/elastalert2#197] [Q&A] [answered]
Hi , as i have deployed elastalert , when i exec into the pod there is no config.yaml in this location /opt/elastalert ,there is only an run.sh and my pod is crashloop because it is not able to the values in from configmap too my set up is as follows required some help[
[github.txt](https://github.com/jertel/elastalert2/files/6558128/github.txt)
[github1.txt](https://github.com/jertel/elastalert2/files/6558129/github1.txt)

Mapping of the config file is not happening

And after exec into the container also i am not able to manually copy the config file it states permission denied , i used official docker image

**Comment by jertel** [comment, marked as answer]
Are you using an old ElastAlert helm chart? The Deployment you linked is incorrect -- It's trying to use a `/opt/config` mount point.

You should review the release notes for 2.1.0 which specifically calls out these path changes: https://github.com/jertel/elastalert2/releases/tag/2.1.0
